apps/package.json

```diff
@@ -117,6 +117,7 @@
     "grunt-webpack": "^1.0.11",
     "hammerjs": "^2.0.8",
     "history": "^2.0.1",
+    "html2canvas": "^0.5.0-beta4",
```

Comment:
Actually, it looks like their last non-beta is quite old, and they've been iterating on betas since beginning of 2015, so using this is likely the right decision.